packages/core/src/messages/challenge.ts

```diff
@@ -0,0 +1,61 @@
+import { u8aEquals, SECP256K1 } from '@hoprnet/hopr-utils'
```

Comment:
should probably be moved to utils alongside the other types.

Reply:
SECP256K1 contains only constants relevant for secp256k1

Reply:
No, this Challenge type, should get moved into utils alongside the others.

Reply:
What do you mean here? The challenge class itself or the secp256k1 constants?

Reply:
the class

Reply:
I disagree because the functionality is highly specific to the PoR scheme and should not be reused without further considerations. Therefore it does not make any sense to have this class in `utils`.